streamingvisitors/src/vespa/searchvisitor/searchvisitor.cpp

```diff
@@ -443,6 +445,12 @@ SearchVisitor::init(const Parameters & params)
                             i, j, string(prop.key(j)).c_str(), string(prop.value(j)).c_str());
                         _rankController.getFeatureOverrides().add(prop.key(j), prop.value(j));
                     }
+                } else if (prop.name() == "highlightterms") { // pick up feature overrides
```

Comment:
Typo, change to: "pick up highlight terms"

Reply:
Comment removed.